weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -1050,8 +1050,8 @@ MyApplet.prototype = {
       case 37:/* isolated thunderstorms */
         return _('Isolated thunderstorms')
       case 38:/* scattered thunderstorms */
-      case 39:/* scattered thunderstorms */
         return _('Scattered thunderstorms')
+      case 39:/* scattered thunderstorms */
```

Comment:
I'm not sure I understand what to do. You should know I'm not really a developer. Slowly learning programming, but not yet there.